pkg/registry/job/strategy.go

```diff
@@ -163,6 +165,23 @@ func JobToSelectableFields(job *batch.Job) fields.Set {
 	return generic.MergeFieldsSets(objectMetaFieldsSet, specificFieldsSet)
 }
 
+func CheckJob(r *registry.Store, ctx api.Context, err error, obj runtime.Object) error {
```

Comment:
Yes, they'reall in list. But still it would be more user-friendly message pointing out the solution of collision, just delete previouse as it is done. Why not print it?